provider/keycloak_openid_client.go

```diff
@@ -55,6 +55,26 @@ func resourceKeycloakOpenidClient() *schema.Resource {
 				Computed:  true,
 				Sensitive: true,
 			},
+			"standard_flow_enabled": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
+			},
+			"implicit_flow_enabled": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
+			},
+			"direct_access_grants_enabled": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
```

Comment:
Much better than Keycloak's default.

Reply:
I also changed the default for the standard flow to false as well, thoughts on that?  I figured this made sense because the standard flow requires at least one valid redirect uri and I didn't want users to get validation errors with a config that doesn't specify either

Reply:
I lean towards having it on by default, since it's the generally recommended flow (although a lot of things use the hybrid flow, which is probably why implicit is on by default). It's also more useful than creating a client with no available flows as the default.

Reply:
I gave it some thought and I still think I'm going to turn it off by default.  My issue is that I don't want a resource like this failing due to not specifying redirect uris:

```hcl
resource "keycloak_openid_client" "client" {
    realm_id = "test"
    client_id = "test"
}
```

The obvious downside is that the example above is pretty much useless with every flow turned off, but I like the idea of being explicit anyways and it anyone who runs into this should quickly figure out the issue.  I can also make a point to call this out in the docs.